README.md

```diff
@@ -105,50 +107,53 @@ Use [cron-parser](https://github.com/harrisiirak/cron-parser).
 └───────────────────────── second (0 - 59, optional)
 ```
 
-example:
+Example:
+
 ```javascript
-// exce task every 3 hours
+// To execute task every 3 hours
 exports.schedule = {
   type: 'worker',
-  cron: '0 0 */3 * * *',
+  cron: '0 0 */3 * * *'
 };
 ```
 
 ### Interval-style Scheduling
 
-Use `setInterval`, and support [ms](https://www.npmjs.com/package/ms) conversion style
+To use `setInterval`, and support [ms](https://www.npmjs.com/package/ms) conversion style
+
+Example:
 
-example:
 ```javascript
-// exce task every 3 hours
+// To execute task every 3 hours
 exports.schedule = {
   type: 'worker',
-  interval: '3h',
+  interval: '3h'
 };
 ```
 
 ### Schedule Type
+
 Build-in support is:
-  - worker: will execute in one random worker when schedule executed.
-  - all: will execute in all workers when schedule executed.
 
-You can extend it by: 
+  - **worker**: will be executed in one random worker when schedule run.
+  - **all**: will be executed in all workers when schedule run.
+
+**Custom schedule by extending the base schedule**
```

Comment:
custom schedule is not extend `worker` or `all`,  just add a new one, not inheritance relationship